downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -78,6 +78,15 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
 
 ### New features
 
+#### `soap` Package
+- Introduced the `soap` standard library package, which provides a client API to connect to SOAP enpoints.
```

Comment:
```suggestion
- Introduced the `soap` standard library package, which provides a client API to connect to SOAP endpoints.
```
Is it correct to use both `library` and `package` words? Should it be the `soap` package?